adoc/Release-Notes.adoc

```diff
@@ -31,6 +31,112 @@ endif::[]
 [id='sec.major-change']
 == Major Changes
 
+[id='sec.1_4']
+=== Release 1.4, May 2019
+
+[id='sec.1_4.new']
+==== What Is New?
+
+* {scf} has been updated to version 2.16.4:
+** A tech preview of Eirini is available.
```

Comment:
Link to https://github.com/SUSE/scf/wiki/Eirini from here as well (not just from known issues).

Reply:
Done in https://github.com/SUSE/cap-release-notes/pull/61